frontend/mr-admin-flate/src/pages/avtaler/AvtaleInfo.tsx

```diff
@@ -180,7 +180,10 @@ export function AvtaleInfo() {
           {avtale?.arenaAnsvarligEnhet ? (
             <Bolk>
               <div style={{ display: "flex", gap: "1rem" }}>
-                <Metadata header="Ansvarlig enhet fra Arena" verdi={avtale.arenaAnsvarligEnhet} />
+                <Metadata
```

Comment:
Hva med en sjekk først, så slipper vi nullability-operator og overskift uten navn for avtaler opprettet utenfor arena? :D  
```
{avtale.arenaAnsvarligEnhet && (
  <Metadata
    header="Ansvarlig enhet fra Arena"
    verdi={`${avtale.arenaAnsvarligEnhet.enhetsnummer} ${avtale.arenaAnsvarligEnhet.navn}`}
  />
)}
```

Reply:
Det er en sjekk på linje 180?

Jeg ser at jeg kan ta bort `?.` så det ordner jeg 👍